python/lsst/afw/table/base/base.cc

```diff
@@ -103,11 +103,13 @@ PyBaseRecord declareBaseRecord(py::module &mod) {
     declareBaseRecordOverloads<double>(cls, "D");
     declareBaseRecordOverloads<float>(cls, "F");
     declareBaseRecordOverloads<lsst::afw::table::Flag>(cls, "Flag");
+    declareBaseRecordOverloads<std::uint8_t>(cls, "B");
```

Comment:
I really think we should add some kind of type trait at some point to standardize the suffixes. Such that we can say `lsst::utils::default_suffix<std::uint8_t>::value` and always get the same one.
Maybe not now, since it leads to the slightly tricky problem of compile-time string concatenation, but perhaps file a new ticket for it? If you agree that is.

Reply:
I agree this code would benefit from a cleanup pass. But I'm not comfortable enough with type traits to have an opinion on this exact issue.

Reply:
This is another issue that is worth doing as a separate ticket.